### DISCUSSION TITLE
[FR] Quick Edit Entry Modal

### DISCUSSION THREAD

**Discussion by Mosnar** [wbrowar/craft-admin-bar#35] [Ideas]
This is an idea I had that I was going to implement in its own plugin, but I thought this might be a better place for it. 

The idea is to allow editing of content inline on the page, but rather than implementing a custom UI, we could do one of the following:
- Load a modal of a particular entry's edit screen in an iframe on the frontend
- Load the entry's "quick edit" dialog on the frontend -  this may also need to be in an iframe to prevent style contamination

**Comment by wbrowar** [comment]
Hey @Mosnar, I like the idea and I have some plans to refactor the plugin a bit so I might not get to it right away, but I'll explore this then.

In the mean time, there is this feature: https://github.com/wbrowar/craft-3-adminbar#admin-bar-widgets

Since the Widgets idea hasn't really taken off (at least to my knowledge), I think that will change a bit in the future, but that's a way to make this happen right now, even though it requires another plugin.
